src/plugins/wizard/public/application/components/data_tab/secondary_panel.tsx

```diff
@@ -56,18 +59,27 @@ export function SecondaryPanel() {
     (isValid: boolean) => {
       // Set validity state globally
       dispatch(
-        setValid({
+        setValidity({
           key: EDITOR_KEY,
           valid: isValid,
         })
       );
+    },
+    [dispatch]
+  );
 
-      // Autosave changes if valid
-      if (valid) {
+  // Autosave is agg value has changed and edits are valid
+  useDebounce(
+    () => {
+      if (isEditorValid) {
```

Comment:
Should we check if isTouched?

Reply:
`isTouched` is a weird flag that makes more sense in the legacy editor. It does not actually tell us if the editor was touched or not. Rather, it tells us if an invalid field was touched or not after we have detected that it is invalid. That isnt useful for the auto save though since we only want to know if the value has changed and if the fields are valid. we can tell that the value has changed based on the dependency array for the `useDebounce` function and if its valid based on the `isEditorValid` flag.

I'm setting `touched` to true if its not valid so that the editor displays the invalid field indicator. Dont ask me why, but thats how the `DefaultEditorAggParams` wants the prop :/